cymem/cymem.pyx

```diff
@@ -43,15 +69,21 @@ cdef class Pool:
         size (size_t): The current size (in bytes) allocated by the pool.
         addresses (dict): The currently allocated addresses and their sizes. Read-only.
         pymalloc (PyMalloc): The allocator to use (default uses PyMem_Malloc).
+        pycalloc (PyCalloc): The allocator to use (default uses PyMem_Calloc).
+        pyrealloc (PyRealloc): The allocator to use (default uses PyMem_Realloc).
         pyfree (PyFree): The free to use (default uses PyMem_Free).
     """
 
     def __cinit__(self, PyMalloc pymalloc=Default_Malloc,
+                  PyCalloc pycalloc = Default_Calloc,
```

Comment:
This is backwards incompatible --- a user who was instantiated the class with something like `mem = Pool(pymalloc=Default_Malloc, pyfree=Default_Free)` would suffer segfaults.

When adding new arguments to a function with defaults, the new arguments need to be placed after the current ones.